extension/tsconfig-inpage-create-injection-script.json

```diff
@@ -6,8 +6,8 @@
 		"outDir": "app/inpage/js/",
 		"rootDir": "app/inpage/ts/",
 		"noEmit": false,
-		"sourceMap": true,
-		"inlineSources": true,
+		"sourceMap": false,
+		"inlineSources": false,
```

Comment:
While we do delete this file after creating, having the source maps available makes troubleshooting significantly easier in the case where you need to debug a failure.  My guess is that the added cost of generating the source map is less than the cost of having a poor debugging experience in the few cases where you actually need to debug.

Of course, you could just turn source maps back on in those cases...